block/initchain.go

```diff
@@ -32,3 +40,21 @@ func (m *Manager) RunInitChain(ctx context.Context) error {
 
 	return nil
 }
+
+// ValidateGenesisBridgeData validates the genesis bridge data from
+// InitChainResponse against the rollapp genesis stored in the hub.
+func (m *Manager) ValidateGenesisBridgeData(dataBytes []byte) error {
+	if len(dataBytes) == 0 {
```

Comment:
not 100% sure, but i think we shouldnt allow this, otherwise we may fail to detect genesis bridge issues in some cases. i guess rdk could return valid data even in mock scenarios by setting the genesis with the required info.

Reply:
done